### DISCUSSION TITLE
osu!lazer not adhering to my tablet driver settings.

### DISCUSSION THREAD

**Discussion by VoxArts** [ppy/osu#23128] [Q&A]
I decided to try out lazer for the first time and the way it works with tablets compared to stable is different and has made it unplayable for me. I have an XP-Pen Deco 01 V2 and I use it for more than just osu. I don't really know why but the upper half of that tablet in osu tends to skip over pixels/it jitters so I've set up a place on the bottom of the tablet on my driver where that issue doesn't happen and I've had zero issues playing on stable. On lazer I can't get my config to work. If I have the tablet setting enabled it would set it's own playable area for my tablet even If I set the whole area of my tablet on both lazer and drivers to be playable it still wouldn't utlize the full area. Turning off the tablet setting in lazer with either full playable area or my osu playable area with drivers would just pin the cursor to the right side of the screen. This makes lazer unplayable with any setting I've tried so far.

**Comment by Rekunan** [comment]
Have you tried disabling the tablet driver and using only lazer's built-in one?

**Reply by VoxArts** [reply]
I just did and I also reset it back to full area. Ill attach an image, the pen size from top to bottom is roughly the "full area" the game gave me. It's still on the top side of the tablet. Making the size smaller just goes around that "full area" so it's practically useless for me. 
https://gyazo.com/34d949b7aca834d7c93be65e285ed081
Here's a link to a image I took for reference.

**Reply by Rekunan** [reply]
Out of curiosity, can you try these things?

- Having only 1 monitor
- Changing the main monitor
- Moving osu to the other monitor

Also, have you checked to make sure mouse sens is at 1.00x, high precision mouse is disabled, and that your mouse drivers are updated?

**Reply by VoxArts** [reply]
The 3 points made no difference when I tried it. I did have high precision mouse enabled but disabling it made no difference. My mouse drivers and tablet drivers are also up to date.

**Reply by Rekunan** [reply]
If ur still experiencing the issue, can you try changing ur ingame area to full again and then sending the log files (#15188)?

**Reply by VoxArts** [reply]
Since I made the thread I've always had it set as full.
[logs.zip](https://github.com/ppy/osu/files/11174208/logs.zip)

These logs are the only logs I know of that are for osu.

**Reply by VoxArts** [reply]
So I recently tried using a different driver. The OpenTabletDriver. I had the same issue with the driver as well. It only recognized the top left of the tablet as it's full work area. It seems that it isn't an Osu issue but rather the tablet is playing goofy with everything other than the XP pen drivers. I have no idea how to fix that.

**Reply by Rekunan** [reply]
Seems like you should ask in the OpenTabletDriver [discord](https://opentabletdriver.net/Discord) instead.

**Reply by VoxArts** [reply]
I forgot to close the discussion. I had issues with the tablet itself. I already returned it and bought wacom one instead. No issues yet.

**Reply by Rekunan** [reply]
Ah ok, amkes sense.
